client/src/komponenter/Infographic/datauthenting.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  AggregertStatistikkDto,
+  Statistikkategori,
+  StatistikkDto,
+} from "../../integrasjoner/aggregert-statistikk-api";
+import { InfographicData } from "./Infographic";
```

Comment:
Importen må fikses for å kompilere.

Reply:
Veldig rart, dette funker fint lokalt

Reply:
`info  - Checking validity of types...
Failed to compile.

./src/Infographic/datauthenting.ts:5:33
Type error: Cannot find module './Infographic' or its corresponding type declarations.

  3 |   Statistikkategori,
  4 | } from "../integrasjoner/aggregert-statistikk-api";
> 5 | import { InfographicData } from "./Infographic";
    |                                 ^
  6 | 
  7 | export function hentUtInfographicData(
  8 |   data: AggregertStatistikkDto
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn run build' returned a non-zero code: 1
Error: Process completed with exit code 1.`

Reply:
Ahh, var blitt noe krøll i det jeg flytta rundt på noen filer. Fikser nå! Takk for gode tilbakemeldinger